src/ui/public/index_patterns/index.d.js

```diff
@@ -0,0 +1,20 @@
+/*
```

Comment:
This file should be named `.d.ts` not `.d.js`

Reply:
the `.d.ts` version already existed. not sure how the compiled version got generated and committed 😳anyway - removed 🙂